docs/en/sql-reference/functions/other-functions.md

```diff
@@ -924,11 +924,108 @@ Result:
 
 Takes state of aggregate function. Returns result of aggregation (finalized state).
 
-## runningAccumulate {#function-runningaccumulate}
+## runningAccumulate {#runningaccumulate}
 
-Takes the states of the aggregate function and returns a column with values, are the result of the accumulation of these states for a set of block lines, from the first to the current line.
-For example, takes state of aggregate function (example runningAccumulate(uniqState(UserID))), and for each row of block, return result of aggregate function on merge of states of all previous rows and current row.
-So, result of function depends on partition of data to blocks and on order of data in block.
+Takes states of the aggregate function, and for each row of block returns the result of aggregate function on merge of states of all previous rows and the current row.
+
+**Syntax**
+
+```sql
+runningAccumulate(agg_state[, grouping]);
+```
+
+**Parameters**
+
+- `agg_state` — State of the aggregate function. [AggregateFunction](../../sql-reference/data-types/aggregatefunction.md#data-type-aggregatefunction).
+- `grouping` — Grouping key. Optional. The state is reset to zero if the parameter value is changed. It can be any of the [supported data types](../../sql-reference/data-types/index.md) for which the equality operator is defined.
+
+**Returned value**
+
+- A column with values that result from accumulating input states for a set of block lines, from the first to the current line. Result of function depends on partition of data to blocks and on order of data in block.
```

Comment:
- Each resulting row contains a result of the aggregate function, accumulated for all the input rows from 0 to the current position. `runningAccumulate` resets states for each new data block or at the `grouping` value changes.